nautobot/dcim/templates/dcim/softwareimage_retrieve.html

```diff
@@ -0,0 +1,174 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load render_table from django_tables2 %}
+
+
+{% block content %}
```

Comment:
Is there a reason we need to override the entire `content` block instead of just `content_left_page`/`content_right_page`/`content_full_width_page`?

Reply:
`content_full_width_page` appears below `left_page` and `right_page` which means the main panel for this object would be below the `tags` panel and any other panels in `left/right_page`. This is completely copy/pasted just to move `content_full_width_page` above those other 2.

Reply:
Brainstorming, what if we put everything in `left_page` *except* the checksum, which goes at the bottom of the page in `full_width_page`? That's the main one that's problematic as a half-column, right?